commitizen/commands/changelog.py

```diff
@@ -49,6 +49,10 @@ def __init__(self, config: BaseConfig, args):
         self.tag_format = args.get("tag_format") or self.config.settings.get(
             "tag_format"
         )
+        self.template = args.get("template") or self.config.settings.get("template")
+        self.extras = dict(
```

Comment:
is there an example of this `=` usage? I'm a bit confused. I guess it's something like `cz ch -e key=value,key=value`?

Reply:
I extracted the parsing in a regular `argparse.Action` and added some more examples.
I opted for `--extra key=value -e key-value` because the parsing is simpler.
It supported single and double quoted string properly, aka. this works: 
```shell
-e key-value -e single='a quoted value' and -e double="another quoted value"
``` 
and should result in:
```python
namespace["extras"] == {"key": "value", "single": "a quoted value", "double": "another quoted value"}
```

Reply:
got it. it makes sense!